Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,309 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace test1
+{
+    public class Program
+    {
+        const int minimo_stock = 1000;
+        static void Main(string[] args)
+        {
+            List<Cliente> arrayclienteNuevo = new List<Cliente>();
+            List<Producto> arrayproductoNuevo = new List<Producto>();
+            List<Vendedor> arrayvendedorNuevo = new List<Vendedor>();
+            List<Venta>    arraynuevaVenta = new List<Venta>();
+
+            int optionMenu = 0;
+            bool answerMenu ;
+            int numberOpcion;
+            while (optionMenu != 6)
+            {
+                do
+                {
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("Menu de Facturacion");
+                    Console.WriteLine("--------------------");
+                    Console.WriteLine("1. Registro de Ventas");
+                    Console.WriteLine("2. Registro de Clientes");
+                    Console.WriteLine("3. Registro de vendedores");
+                    Console.WriteLine("4. Registro de Productos");
+                    Console.WriteLine("5. Reportes");
+                    Console.WriteLine("6. Salir");
+                    Console.WriteLine("Seleccione una opcion");
+                    answerMenu= Int32.TryParse(Console.ReadLine(), out numberOpcion);
+                }   while (answerMenu==false);
+                optionMenu = numberOpcion;
+                switch (optionMenu)
+                {
+                    case 1: RegistroVentas(arraynuevaVenta); break;
+
+                    case 2: RegistroClientes(arrayclienteNuevo); break;
+
+                    case 3: RegistroVendedores(arrayvendedorNuevo); break;
+
+                    case 4: RegistroProductos(arrayproductoNuevo); break;
+
+                    case 5:                
+                        {
+                            int opcion = 0;
+                            int answer = 0;
+                            bool boolAnswer = true;
+                            while (opcion != 5)
+                            {
+                                do
+                                {
+                                    Console.WriteLine("--------------------");
+                                    Console.WriteLine("Menu de Reportes");
+                                    Console.WriteLine("1. Total de Ventas");
+                                    Console.WriteLine("2. Clientes mayores a 55 años de edad");
+                                    Console.WriteLine("3. Clientes que cumplen años en el mes");
+                                    Console.WriteLine("4. Productos con stock bajo.");
+                                    Console.WriteLine("5. Regresar.");
+                                    Console.WriteLine("Seleccione una opcion");
+                                    boolAnswer = Int32.TryParse(Console.ReadLine(),out answer);

```

Comment:
but if i declare the variable 'answer' at the same what send as out argument (out int answer), the variable 'answer' does will not exist in the current text, because it used out side the bucle do while.